app/models/inning.rb

```diff
@@ -45,4 +75,17 @@ def bottom_of_previous_inning_exists
         status: Inning.statuses["completed"]).
       count == 1
   end
+
+  def update_game_score
+    if half == "top"
+      game.update(away_score: game.innings.where(half: Inning.halves["top"]).sum(:runs))
+    else
+      game.update(home_score: game.innings.where(half: Inning.halves["bottom"]).sum(:runs))
+    end
+  end
+
+  def update_status
+    # TODO: What about if game ends with < 3 outs?!?
```

Comment:
Good point...we might want to ensure that we're handling this. I'm guessing we are not currently doing so.